crates/sncast/src/helpers/constants.rs

```diff
@@ -30,3 +30,5 @@ pub const CREATE_KEYSTORE_PASSWORD_ENV_VAR: &str = "CREATE_KEYSTORE_PASSWORD";
 
 pub const SCRIPT_LIB_ARTIFACT_NAME: &str = "__sncast_script_lib";
 pub const CONFIG_FILENAME: &str = "snfoundry.toml";
+
+pub const SCRIPTS_DIR: &str = "scripts";
```

Comment:
```suggestion
pub const INIT_SCRIPTS_DIR: &str = "scripts";
```
maybe, to avoid confusion? technically scripts dir can be anything, and this is only used for init